geth/node/manager.go

```diff
@@ -32,15 +29,20 @@ var (
 	ErrRPCClient                   = errors.New("failed to init RPC client")
 )
 
+// RPCClientError reported when rpc client is initialized.
+type RPCClientError error
+
+// EthNodeError is reported when node crashed on start up.
+type EthNodeError error
+
 // NodeManager manages Status node (which abstracts contained geth node)
 // nolint: golint
 // should be fixed at https://github.com/status-im/status-go/issues/200
 type NodeManager struct {
-	sync.RWMutex
-	config         *params.NodeConfig // Status node configuration
-	node           *node.Node         // reference to Geth P2P stack/node
-	nodeStarted    chan struct{}      // channel to wait for start up notifications
-	nodeStopped    chan struct{}      // channel to wait for termination notifications
+	mu     sync.RWMutex
```

Comment:
In the next iteration, maybe it would be possible to also remove `NodeManager` and have only `Node`.